setup.cfg

```diff
@@ -9,17 +9,6 @@ upload = upload upload_docs --upload-dir build/sphinx/html
 [bdist_rpm]
 doc-files = doc
 
-[nosetests]
```

Comment:
Yes, @lesteve I don't mind the noise anyways.. Actually I am enjoying this discussion for getting the PR merged. :four_leaf_clover: